src/jdk.compiler/share/classes/com/sun/tools/javac/api/JavacTrees.java

```diff
@@ -624,12 +624,10 @@ private Symbol attributeParamIdentifier(TreePath path, DCParam paramTag) {
         return null;
     }
 
-    /** @see com.sun.tools.javadoc.ClassDocImpl#findField */
```

Comment:
Is there an equivalent new-style method that should be used instead?

Reply:
The most likely justification for all these ClassDocImpl references is now in `ReferenceTree`.

Reply:
A doc comment consisting solely of an `@see` tag has unclear semantics. The methods I removed such doc comments from are private and can be used only from their enclosing instance.

Reply:
Let me go do some archaeology. While the comments may have been substandard, I would still like to recall their intent.

Reply:
The notable content of the comments on those `ClassDocImpl` methods is that they document (I hesitate to use the "specify" word) the search order, with the additional comment *Note that this is not necessarily what the compiler would do!*